### ISSUE TITLE
how to train custom model for other country license plate

### ISSUE THREAD

**Issue by RajMaster1612** [xuexingyu24/License_Plate_Detection_Pytorch#19]
how i train this with my own dataset form scratch

**Comment by pendex900x** [comment]
I have the same doubt

**Comment by danielchang1985** [comment]
exactly. I tried running with other datasets(e.g. license plate with english alphabets only) but it can't capture the shape in the bboxes. The above script only works for chinese license plates.
